release-notes/release-5.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: April 17, 2024
+---
+
+# Features
+
+- [Oracle Database Integration]() with default dashboard
+- [Apache Networking Integration]() with default dashboard
+- Automated log collection with the Vercel APM for __Build__, __Lambda__, __Static__, __Edge__, and __External__ immediately upon installation
+- Ability to parse attributes from non-standard JSON log messages
+- Improved sharing of RUM links with filters as query parameters within RUM to easily share user-specific sessions with your teammates
+- Users can set alerts based on their usage and budget on daily or monthly intervals
+- Advanced settings added to alerts. Users can configure alerts that will trigger when a certain specified threshold X is surpassed by Y number of occurrences over a given time interval Z
+- Updated filter sidebar in the APM Page allows users to search across all of their instrumented application data
+- Dashboards can be dynamically generated based on a single metric that is being collected by Middleware
+- Filter the Span List to see a deeper view of all spans related within the APM Dashboard by parent `SpanID` to quickly identify what you are looking for
+- Added __Critical__ severity level message that can be auto-detected across all logs
+- Database Monitoring query filter can now sort queries by __Most Time Consuming__, __Slowest Query Time__, and __Throughput__
+
+# Changes
+
+- Redesigned RUM dashboard for simplicity
+- Added a prompt when deleting projects to prevent accidental data loss
+- Dashboard Builder has a fully revamped dashboard and widget creation workflow that increases the speed by which you create new dashboards and widgets
+- More efficient flame graph processing in the APM dashboard
+- When users change their page from History mode to Live mode, Live mode now only displays the last 5 minutes instead of the last hour
```

Comment:
Remove the "when users change their page from History mode to Live mode", as it doesn't matter how they get to live, live will always only be the last 5 minutes.